README.rst

```diff
@@ -141,6 +141,28 @@ Project resources
 - `Download development snapshot <https://github.com/mopidy/mopidy-spotify/tarball/develop#egg=Mopidy-Spotify-dev>`_
 
 
+Helping out
+===========
+
+The following commands should get you going.::
+
+    git clone https://github.com/mopidy/mopidy-spotify.git
+    cd mopidy-spotify
+
+    # start editing files
+
+    # install your changes
+    sudo python setup.py install
```

Comment:
You normally want `python setup.py develop` and not install, as develop will create a pointer to your dev dir so you don't have to re-install for every change. I would also recommend doing things in a virtualenv and not globally, but that might be out of scope for such a getting started guide.